acme/acme/challenges_test.py

```diff
@@ -359,9 +359,12 @@ def test_simple_verify(self, mock_verify_cert):
             cert=mock.sentinel.cert))
         mock_verify_cert.assert_called_once_with(self.msg, mock.sentinel.cert)
 
-    def test_simple_verify_false_on_probe_error(self):
+    @mock.patch('acme.challenges.socket.gethostbyname')
+    def test_simple_verify_false_on_probe_error(self, mock_gethostbyname):
         chall = mock.Mock()
         chall.probe_cert.side_effect = errors.Error
+
+        mock_gethostbyname.return_value = '127.0.0.2'
```

Comment:
I think another PR is a better approach, but I honestly don't really care too much since only a one line change is necessary. See my comments on #1183 or Kuba's original comment in this thread.